swagger_spec_validator/validator20.py

```diff
@@ -285,10 +285,10 @@ def validate_unresolvable_path_params(path_name, path_params):
 
     :raises: :py:class:`swagger_spec_validator.SwaggerValidationError`
     """
-    msg = "Path Parameter used is not defined"
+    msg = "Path Parameter used is not documented"
     for path in get_path_params_from_url(path_name):
         if path not in path_params:
-            raise SwaggerValidationError("%s: %s" % (msg, path))
+            raise SwaggerValidationError("%s: %s in %s" % (msg, path, path_name))
```

Comment:
To make the message even more explicative I would change the error message in something similar to ``Path parameter '{path}' used is not documented on {path_name}``